CPS-0004/README.md

```diff
@@ -0,0 +1,96 @@
+
+For a variety of reasons, including moral, philosophical, financial, and possibly regulatory, these founding entities want to dilute their own power substantially, and share the burden of governance of the chain with the community that has arisen around it.
+
+IOG in particular has continued to provide skill and resources for years longer than initially intended, and is significantly over-budget and over-extended. IOG has said that they may be unable or unwilling to support development of the chain indefinitely, and so an expedient and iterative approach to governance is likely to have higher positive impact on our ability to maintain velocity as an ecosystem.
+
+## **Goals**
+
+Synthesizing from the above, I posit these (hopefully uncontroversial) goals, constraints, considerations, or ground truths for any governance system. I'm only one person, though, so this list will very likely be revised and updated by collective community input as discussion unfolds.
+
+1. A system of on-chain rules should allow decisions to be made and enacted regarding (at a minimum) the three types of changes listed above.
+2. Every (or nearly every) ADA holder should be able to meaningfully participate in this decision making process.
+3. This system should not materially undermine the security of the chain.
+4. This system should not undermine it's ability to serve it's financial function, such as overburdening the chain, damaging the monetary policy, etc.
+5. This system should be implementable within a reasonable time frame and budget.
+6. This system should be recursively updatable; if improvements or changes to the system of governance are needed, they should be voted on and enacted via this system.
+7. This system is highly dependent on the off chain user experience and community tooling, since very few users are reading raw CBOR dumps from the chain.
+8. This system should be cognizant of the fact that different decisions can have different impact and risk thresholds.
+9. This system cannot avoid the fact that stake pool operators of the network, collectively, always maintain control over a "pocket nuke": under sufficient discontent and misalignment with the will of governance, they can modify the code and initiate an uncontrolled hard fork to produce a separate chain aligned with their values.
+
+Additionally, here is a more opinionated (and potentially controversial) list of potential considerations for a CIP, presented purely to foster discussion:
+
+1. There is currently no known decentralized "proof of personhood" system that is well specified and easily implementable, and without such a system, "one lovelace one vote" systems are the most effective defense against sybil attacks.
```

Comment:
https://go.gitcoin.co/passport is one such system

Reply:
I couldn't find an explanation of how it works beyond just being a standard for different centralized entities to issue "stamps"; if that is the case, who manages the "stamp issuance" for being able to participate in a vote? That's what I meant by "decentralized", but maybe I can emphasize that more.

Reply:
Adding "This system cannot rely on a trusted third party. " as a core goal would help making the "proof of personhood" point less controversial (maybe? 🥲) . It makes it much clearer to the general public why 1 lovelace 1 vote is the best we can do currently.

Reply:
From my understanding it looks to be a protocol where multiple dapps can issue 'stamps' (attestations) in your 'passport' (agreed on data format), which is basically a decentralized way to build up proof of unique humanness. Each dapp can then decide what combination of stamps sufficiently proves this. This is used as a weight on quadratic funding for gitcoin.

I squint and see PGP's web of trust.

I don't know how much of this is available or portable to Cardano.

I'm not saying 1 ADA = 1 vote is not the way to go, but it's interesting that other chains have chosen to tackle this for governance for community grants

Reply:
That's exactly what Atala Prism is, fwiw. And works great for specific limited domains. The mind balks at getting Cardano to agree to a specific set of stamps that qualify you though 😅 especially when we haven't even *started* laying that groundwork outside of the standard. There's not a single DID issuer with a single day of reputation yet.

Reply:
:100:  :point_up: